### PR TITLE
feat(archive): Catch random errors in BoxArchive and throw preview error

### PR DESCRIPTION
**Pull request by mxiao6**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @mxiao6 has signed the CLA. Thanks for the pull request!

File: src/lib/viewers/archive/ArchiveViewer.js [hunk 1]
```diff
@@ -68,11 +69,12 @@ class ArchiveViewer extends BaseViewer {
...
 
-        /*
-            global BoxArchive
-            The BoxArchive is loaded from archive.js
-        */
-        this.archiveComponent = new BoxArchive(this.archiveEl, data);
+        try {
+            /* global BoxArchive loaded from archive.js */
+            this.archiveComponent = new BoxArchive(this.archiveEl, data);
+        } catch {
```
Comment (mickr) [inline src/lib/viewers/archive/ArchiveViewer.js]: Are there not more descriptive errors thrown to the catch statement?

Comment (jstoffan) [reply src/lib/viewers/archive/ArchiveViewer.js]: Can refreshing resolve the issue for the user? Are these client-side issues or data issues?

Comment (mxiao6) [reply src/lib/viewers/archive/ArchiveViewer.js]: > Can refreshing resolve the issue for the user? Are these client-side issues or data issues?

Refreshing probably cannot resolve the issue 😂. The issue maybe React issue or data issues.

**Pull request merged**